### ISSUE TITLE
Cannot connect to roborock server anymore

### ISSUE THREAD

**Issue by RaoulV82** [humbertogontijo/homeassistant-roborock#172]
Hi I'm not sure if this is related to this integration or another issue but noted the latest release update notes about retrying connections. I can no longer connect to the roborock servers via the phone app or other devices whilst using my home WiFi/ broadband. I seem to be able to connect using 4g or someone else's broadband.
I had turned off my vacuum over the weekend but HA may have been trying to connect to it as I didn't turn off the integration.  It is possible I've been barred in someway from the server and can test this?
Thanks for your help.

**Comment by humbertogontijo** [comment]
Can you check your logs for reconnections?

**Comment by RaoulV82** [comment]
Hi humbertogontijo, I'm not use how to check historic logs, are you able to advise. HOWEVER, the good news is I've managed to solve the issue. I figured that given I can still log in with my creds using any connection other than my broadband, if they had barred me it would be based on my external IP. I forced a change in my external IP and can now connect back to the app so does seem to suggest my IP was banned. Looks like your latest update may already solve for this (of course if that was the issue!).
